README.md

```diff
@@ -1,6 +1,6 @@
 # Project Initializer
 
-[![Build Status](https://semaphoreci.com/api/v1/fabric8-launcher/launcher-vscode-extension/branches/master/badge.png)](https://semaphoreci.com/fabric8-launcher/launcher-vscode-extension)
+[![Build Status](https://travis-ci.com/fabric8-launcher/launcher-vscode-extension.svg?branch=master)](https://travis-ci.com/fabric8-launcher/launcher-vscode-extension)
```

Comment:
What's the rationale of using travis-ci.com instead of travis-ci.org ? Because the .com link says 'Build unknown' where the .org link says 'Build passed'
Also your PR has been rejected by DCO check

Reply:
Travis-ci.org will be discontinued in the future. New Users are asked to migrate to travis-ci.com (https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps)

It says Build Unknown because no build was run yet in the new environment (because migrating to .com does not bring the build history)